README.md

```diff
@@ -1,7 +1,172 @@
-# VisualMQ
 
-## Development
-1. Clone this repo.
-1. Open the folder in Unity as a new project.
-1. Create and develop :art:
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![License][license-shield]][license-url]
 
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="https://github.com/VisualMQ/VisualMQ/">
+    <img src="visualmq-logo.png" alt="Logo">
```

Comment:
I would make the image smaller (changing `width` or `height` HTML attributes)

Reply:
The image needs modification anyhow. Not visible on dark-mode 
![image](https://user-images.githubusercontent.com/33268295/129859871-81173eb5-a1be-4345-bb2a-9f0aa72d5428.png)

Reply:
![image](https://user-images.githubusercontent.com/33268295/129859903-1949c104-c9a5-4c00-973f-502e7499792b.png)